docs/index.md

```diff
@@ -21,25 +21,49 @@ DDEV by default uses ports 80 and 443 on your system when projects are running.
 
 ## Installation
 
-### Docker Installation
+### Docker or Colima Prerequisite
 
 Docker or an alternative is required before anything will work with DDEV. This is pretty easy on most environments; see the [docker_installation](users/docker_installation.md) page to help sort out the details, especially on Windows and Linux. It is not required to install docker-compose because DDEV uses its own private version.
 
-### Homebrew - macOS/Linux
+### macOS (Homebrew)
 
-For macOS (both amd64 and arm64) and Linux users, we recommend installing and upgrading via [Homebrew](https://brew.sh/) (macOS) or [Homebrew on Linux](https://docs.brew.sh/Homebrew-on-Linux) (Linux): `brew install drud/ddev/ddev`.
+For macOS (both amd64 and arm64) users, we recommend installing and upgrading via [Homebrew](https://brew.sh/): `brew install drud/ddev/ddev`.
 
-As a one-time initialization, run `mkcert -install`. Linux users may have to take additional actions as discussed below in [Linux `mkcert -install` additional instructions](#Linux-mkcert--install-additional-instructions).
+As a one-time initialization, run `mkcert -install`.
 
 Later, to upgrade to a newer version of DDEV-Local, run `brew upgrade ddev`.
 
-To install the latest unreleased DDEV version, `brew unlink ddev && brew install drud/ddev/ddev --HEAD`.
+To install DDEV prereleases, subscribe to the "edge" channel with `brew install drud/ddev-edge/ddev` and to install the latest unreleased DDEV version, `brew unlink ddev && brew install drud/ddev/ddev --HEAD`.
 
-### Installation or Upgrade - Windows (WSL2)
+### Linux (Arch)
 
-**This is the recommended installation method for all Windows users that are on Windows 10 2004 or higher**. If you don't have this version yet, or if you don't want to use WSL2, please follow the legacy instructions for Windows below.
+We maintain a package on [Arch Linux (AUR)](https://aur.archlinux.org/packages/ddev-bin/).
 
-**All Windows 10 editions (including Windows 10 Home) support WSL2**. Docker Toolbox support for DDEV has been removed. If you're already familiar with DDEV on Windows, you might have been using NFS for better filesystem performance. **You won't need NFS anymore once you switch to WSL2**, since it provides awesome filesystem performance out of the box.
+As a one-time initialization, run `mkcert -install`, which may require your sudo password. See below for additional information.
```

Comment:
Is this something that could be included with the package install?  You don't have to change anything now.  I was just wondering if it was possible.

Reply:
There have been attempts to do a do-it-all-in-script but they didn't make it to the finish line. It's slightly complicated because you have to do things on both wsl2 side and windows side, but that's do-able. Another attempt would be welcome!